dark/index.html

```diff
@@ -312,7 +313,6 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <!-- Contact Section -->
 
       <section id="contact-section">
-        <h1>CONTACT ME</h1>
         <h3>CONTACT <span class="text-info">ME</span></h3>
         <p class="contact-text">📚 Education : St. Anthony's College, Kandy</p>

```

Comment:
Remove this also we have another @muskaan04

Reply:
@YoshithaRathnayake  updated as per specifications

Reply:
So can you remove the p tag @muskaan04 
```suggestion
        <p class="contact-text">📚 Education : St. Anthony's College, Kandy</p>

```

Reply:
Updated @YoshithaRathnayake